ibm_catalog.json

```diff
@@ -0,0 +1,546 @@
+        {
+          "title": "Virtual server instance",
+          "description": "Creates and configures a Virtual server instance."
+        },
+        {
+          "title": "KMS encryption",
+          "description": "Supports creating a new key, or using an existing one to encrypt the Block storage volume."
+        },
+        {
+          "title": "Load balancer",
+          "description": "Supports creating and configuring Load balancers for the Virtual server instance."
+        },
+        {
+          "title": "Block storage volumes",
+          "description": "Supports creating and configuring Block storage volumes for the Virtual server instance."
+        }
+      ],
+      "flavors": [
+        {
+          "label": "Fully configurable",
+          "name": "fully-configurable",
+          "install_type": "fullstack",
+          "working_directory": "solutions/fully-configurable",
+          "compliance": {
+            "authority": "scc-v3",
+            "profiles": [
+              {
+                "profile_name": "IBM Cloud Framework for Financial Services",
+                "profile_version": "1.7.0"
+              }
+            ]
+          },
+          "configuration": [
+            {
+              "key": "ibmcloud_api_key"
+            },
+            {
+              "key": "prefix",
+              "required": true,
+              "description": "Prefix to add to all resources that this solution creates. To not use any prefix value, you can enter the string `__NULL__`."
+            },
+            {
+              "key": "existing_resource_group_name",
+              "required": true,
+              "custom_config": {
+                "type": "resource_group",
+                "grouping": "deployment",
+                "original_grouping": "deployment",
+                "config_constraints": {
+                  "identifier": "rg_name"
+                }
+              }
+            },
+            {
+              "key": "existing_vpc_crn",
+              "required": true
+            },
+            {
+              "key": "existing_subnet_id",
+              "required": true,
+              "custom_config": {
+                "type": "platform_resource",
+                "grouping": "deployment",
+                "original_grouping": "deployment",
+                "config_constraints": {
+                  "resourceType": "subnet",
+                  "selection": "single_select",
+                  "valueType": "id"
+                },
+                "associations": {
+                  "parameters": [
+                    {
+                      "name": "region",
+                      "optionsRefresh": true
+                    }
+                  ]
+                }
+              }
+            },
+            {
+              "key": "image_id",
+              "required": true
+            },
+            {
+              "key": "existing_secrets_manager_instance_crn",
+              "required": true
```

Comment:
why are we requiring this? I think it should be optional. User can opt into it if they want to store SSH keys in a secret